### ISSUE TITLE
[Slider] double handle : add prop to set active handle

### ISSUE THREAD

**Issue by kevin-krug-sociomantic** [sociomantic-tsunami/nessie#337]
As per concept in order to avoid the upper and lower range values crossing each other, an api for setting which handle is active should be added

see related concept for the requested microbehaviour
https://github.com/sociomantic/ursula/blob/Milestone-UIModels/src/lego/Presenters/Slider/docs/Slider.spec.md#microbehavior

simple suggestion would be adding an `isActiveHandle` prop.

**Comment by conor-cafferkey-sociomantic** [comment]
I don’t think this is going to be possible. At least not with the current Slider implementation where each handle is associated with a hidden native `<input type="range">`.

Problem is, setting the active handle actually implies programmatically focusing the `<input>` associated with the desired handle. But we should never implement behaviors where the Nessie component steals the focus like that.

Marking as `status-blocked` for now, while we find an alternative solution

**Comment by kevin-krug-sociomantic** [comment]
@conor-cafferkey-sociomantic okay, sorry for this request. No need for any Nessie changes, we can just treat e.g the upper as lower handle, if it passes the lower and vice versa, as there's no visual difference.. No dependency!

**Comment by kevin-krug-sociomantic** [comment]
actually as initially discussed swapping values does ensure correct lower and upper range values, however the focus is not swapped!

**Comment by kevin-krug-sociomantic** [comment]
![out](https://user-images.githubusercontent.com/6839359/39372117-d5ae1742-4a43-11e8-85a3-f0a6d1131211.gif)

**Comment by conor-cafferkey-sociomantic** [comment]
We’ll add an `inputRef` prop similar to that on other props to allow the host app to programmatically focus the input(s):

- in the case of a single value/handle the `inputRef` callback will receive a single ref to the native `<input>`
- in the case of multiple values/handles the callback will receive an array of refs
